### PR TITLE
Added accuracy and consistency test to kernel explainer.

### PR DESCRIPTION
**Pull request by jorgecarleitao**
This commit adds testing of KernelExplainer against the properties on the paper (local accuracy and consistency/monotonicity).

Note that this currently does not pass, as the KernelExplainer is not guaranteeing consistency as outlined in the paper. Let's figure out why and act accordingly.

I have validated that the consistency test is correct by writing the brute force version of shapley values (without local approximations), and validating the test against that explainer (with multiple training data and non-linear functions).

### CODE REVIEW

**Comment by slundberg** [comment]
The kernel explainer (given enough samples) is consistent, there is just a typo in here at the moment. Once that's fixed I would love to merge the new test :)

**Comment by jorgecarleitao** [comment]
Great to know, because I have been assuming that the KernelExplainer is consistent and that I made a mistake on the test. :P

**Comment by slundberg** [comment]
I sent an updated file your way, let me know if you need to update this PR

**Comment by slundberg** [comment]
It looks like this is still failing. I thought I remember we had this sorted out though over email. What's the status here? (and thanks again!)
